src/main/java/spotify/factories/AbstractPlayableObjectDeserializer.java

```diff
@@ -7,13 +7,46 @@
 
 import java.lang.reflect.Type;
 
+/**
+ * This is a AbstractPlayableObjectDeserializer class to implement JsonDeserializer.
```

Comment:
This actually just repeats this part `class AbstractPlayableObjectDeserializer implements JsonDeserializer<AbstractPlayableObject>` but doesn't really describe what the class is for. Javadoc should describe what the class does and what it is used for.

Reply:
Hi @jzheng2017 ,

Thanks for your  feedback. Would this description be better?

This is a AbstractPlayableObjectDeserializer class which provides the deserialize method to deserialize json, type and jsonDeserializationContext. Thanks.

Reply:
It's better than what it was before, but not quite. It should explain _what_ the class does without going too much into the implementation. 

So a rough description would be like, `This class is a custom {@link JsonDeserializer} which deserializes json and returns a {@link AbstractPlayableObject} subclass based on the type property in the json `

Reply:
Thanks. I will update it as per your feedback.